src/app/register-page/register-page.component.html

```diff
@@ -1 +1,18 @@
-    <app-register-form></app-register-form>
\ No newline at end of file
+<body>
+    <div class="resized">
+      <ul>
+        <li><a href="#">PRADŽIA</a></li>
+        <li class="active"><a [routerLink]="['/register']">REGISTRACIJA</a></li>
+        <li><a >APIE MUS</a></li>
+      </ul>
+    </div>
+    <div class="title1">
+        <h1>REGISTRACIJA</h1>
+    </div>
+    <div class="text1-container">
+       <p> Užpildykite žemiau esančią aplikacijos formą,<br> norint užsiregistruoti IT akademijai. </p>
+    </div>
+    <div class="register-container">
+        <app-register-form></app-register-form>
```

Comment:
Is the register form only used in the register page? Why not move the form into the register-page folder? Easier to understand the structure that way.